README.md

```diff
@@ -222,6 +223,9 @@ will be available after the `api.js` is loaded. For more information, see [Captc
 
 Please refer to the [guide for using risk-based authentication with the widget](/docs/riskAuthentication.md) for more infomation on how to set up the widget with risk-based authentication adapters.
 
+## Redirectless Support 
+Please refer to the [Redirectless Support](/docs/redirectless.md) for more infomation on how to configure PingFederate and how to use widget's redirectless feature.
+
```

Comment:
Maybe have this say something like refer to the Redirectless Support guide/doc for more...